framework/src/controller/helpers/validator/index.js

```diff
@@ -0,0 +1,83 @@
+const Ajv = require('ajv');
+const { SchemaValidationError } = require('../../../errors');
+const formats = require('./formats');
+
+const validator = new Ajv({
+	allErrors: true,
+	schemaId: 'auto',
+	useDefaults: false,
+	$data: true,
+});
+
+const validatorWithDefaults = new Ajv({
+	allErrors: true,
+	schemaId: 'auto',
+	useDefaults: true,
+	$data: true,
+});
+
+Object.keys(formats).forEach(formatId => {
+	validator.addFormat(formatId, formats[formatId]);
+});
+
+Object.keys(formats).forEach(formatId => {
+	validatorWithDefaults.addFormat(formatId, formats[formatId]);
+});
+
+/**
+ * Function helps with loading and validating schemas.
+ *
+ * @type {{loadSchema: module.exports.loadSchema, validate: (function(*=, *=): boolean)}}
+ */
+module.exports = {
```

Comment:
No unit tests coverage for this logic.

As agreed on the offline meeting this part will be moved into the separate file `framework/src/config.js`.